common/src/objects/agent_map.rs

```diff
@@ -16,10 +16,35 @@ use api::ank_base;
 use serde::{Deserialize, Serialize};
 use std::collections::{hash_map::Entry, HashMap};
 
+use crate::commands;
+
 type AgentName = String;
 
+pub const MULTIPLYING_FACTOR: f32 = 100.0;
+
 #[derive(Debug, Serialize, Deserialize, Clone, Default, PartialEq, Eq)]
-pub struct AgentAttributes {} // empty for now, but used for future expansion
+pub struct CpuLoad {
+    pub cpu_load: u32,
+}
+
+impl CpuLoad {
+    pub fn new(cpu_load: f32) -> Self {
+        Self {
+            cpu_load: (cpu_load * MULTIPLYING_FACTOR).trunc() as u32,
```

Comment:
Adding the constant here was intended to improve readability. The name MULTIPLYING_FACTOR is now hiding what we do. Maybe:
```suggestion
            cpu_load: (cpu_load * PERCENTAGE_BASE).trunc() as u32,
```